simple-mpc-mode.el

```diff
@@ -39,6 +39,7 @@
     (define-key map "c" 'simple-mpc-view-current-playlist)  ;; autoload this
     (define-key map "C" 'simple-mpc-clear-current-playlist)
     (define-key map "S" 'simple-mpc-shuffle-current-playlist)
+    (define-key map "r" 'simple-mpc-toggle-repeat)
```

Comment:
Let's move this under `simple-mpc-decrease-volume` so it stays in the same order as in my other comment.